docs/pages/branding/home.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { ThemeProvider, createTheme } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import AppHeader from 'docs/src/layouts/AppHeader';
+import Hero from 'docs/src/components/home/Hero';
+
+const brandingTheme = createTheme({
+  palette: {
+    primary: {
+      main: '#007FFF',
+    },
+    text: {
+      primary: '#0a1929',
+      secondary: '#5a6978',
+    },
+  },
+  shape: {
+    borderRadius: 10,
+  },
+  spacing: 10,
+  typography: {
+    fontFamily: "'PlusJakartaSans', sans-serif",
+    button: {
+      textTransform: 'initial',
+      fontWeight: 'bold',
+    },
+  },
+  components: {
+    MuiTypography: {
+      defaultProps: {
+        variant: 'body2',
+      },
+    },
+    MuiButton: {
+      defaultProps: {
+        disableElevation: true,
+        disableTouchRipple: true,
+      },
+      styleOverrides: {
+        sizeLarge: {
+          padding: '13px 20px',
```

Comment:
```suggestion
          padding: '16px 20px',
```
Have you tried it? Using odd numbers for that makes me feel weird 😅

Reply:
me too but it is because I use `size=large` and it has different line-height from webflow. I feel it is too much to change a lot of styles at this point. So let's keep this weird number for now.

Reply:
fixed.